concert/storage.py

```diff
@@ -454,8 +439,10 @@ async def exists(self, *paths: str) -> bool:
         """Check if *paths* exist."""
         return os.path.exists(os.path.join(await self.get_current(), *paths))
 
-    async def get_log_handler(self) -> AsyncLoggingHandlerCloser:
-        return LoggingHandler(os.path.join(await self.get_current(), self._log_name))
+    async def register_logger(self, logger_name: str, log_level: int,
+                              file_name: str) -> AsyncLoggingHandlerCloser:
+        # With decentralized concert logging utility is delegated to the remote directory walker
+        return NoOpLoggingHandler()
```

Comment:
copy-paste error? This was `LoggingHandler(os.path.join(await self.get_current(), self._log_name))` before.

Reply:
Reverted.